addons/web/static/tests/webclient/actions/load_state.test.js

```diff
@@ -0,0 +1,1708 @@
+import { beforeEach, describe, expect, test } from "@odoo/hoot";
+import { animationFrame } from "@odoo/hoot-mock";
+import { Component, onMounted, xml } from "@odoo/owl";
+import {
+    contains,
+    defineActions,
+    defineMenus,
+    defineModels,
+    fields,
+    getService,
+    makeMockEnv,
+    models,
+    mountWithCleanup,
+    onRpc,
+    patchWithCleanup,
+    stepAllNetworkCalls,
+    toggleMenuItem,
+    toggleSearchBarMenu,
+} from "@web/../tests/web_test_helpers";
+
+import { browser } from "@web/core/browser/browser";
+import { registry } from "@web/core/registry";
+import { WebClient } from "@web/webclient/webclient";
+import { router, startRouter } from "@web/core/browser/router";
+import { redirect } from "@web/core/utils/urls";
+import { ControlPanel } from "@web/search/control_panel/control_panel";
+import { _t } from "@web/core/l10n/translation";
+import { user } from "@web/core/user";
+import { queryAllTexts } from "@odoo/hoot-dom";
+
+describe.current.tags("headless");
```

Comment:
I wouldn't tag them headless, there're not as they mount stuff in the dom. If you don't want them to be executed in mobile, tag them "desktop" instead...